components/eventing-controller/go.mod

```diff
@@ -4,13 +4,15 @@ go 1.14
 
 require (
 	github.com/avast/retry-go v2.4.1+incompatible
+	github.com/cloudevents/sdk-go/protocol/nats/v2 v2.3.1
```

Comment:
Why not use the same version used here https://github.com/kyma-project/kyma/blob/release-1.24/components/eventing-controller/go.mod#L7?

Reply:
They are different things:
''github.com/cloudevents/sdk-go/v2 v2.4.1"
and 
"github.com/cloudevents/sdk-go/protocol/nats/v2 v2.3.1"

Reply:
"github.com/cloudevents/sdk-go/v2 v2.4.1" is used in both event-publisher-proxy and eventing-controller